tests/test_ops.py

```diff
@@ -1734,9 +1790,241 @@ def test_mathtransform():
+         #{"opName": "bitcast", "outName": "bitcast/from_int32_to_half", "varShapes": [[2]], "varTypes": ["int32"],"varInit": ["uniform_int2"], "output": tf.half},
+         #{"opName": "bitcast", "outName": "bitcast/from_int64_to_half", "varShapes": [[2]], "varTypes": ["int64"],"varInit": ["uniform_int2"], "output": tf.half},
+         #{"opName": "bitcast", "outName": "bitcast/emptyArrayTest/from_uint32_to_half", "varShapes": [[0]],"varTypes": ["uint32"], "varInit": ["empty"], "output": tf.half},
+
+        #{"opName": "bitcast", "outName": "bitcast/from_float32_to_float32", "varShapes": [[2]], "varTypes": ["float32"], "varInit": ["uniform"], "output": tf.float32},
+        #{"opName": "bitcast", "outName": "bitcast/from_bfloat16_to_float32", "varShapes": [[2]], "varTypes": ["bfloat16"], "varInit": ["uniform"], "output": tf.float32},
+        #{"opName": "bitcast", "outName": "bitcast/from_half_to_float32", "varShapes": [[2]], "varTypes": ["half"], "varInit": ["uniform"], "output": tf.float32},
+        #{"opName": "bitcast", "outName": "bitcast/from_float64_to_float32", "varShapes": [[2]], "varTypes": ["float64"], "varInit": ["uniform"], "output": tf.float32},
+        #{"opName": "bitcast", "outName": "bitcast/from_int32_to_float32", "varShapes": [[2]], "varTypes": ["int32"],"varInit": ["uniform_int2"], "output": tf.float32},
+        #{"opName": "bitcast", "outName": "bitcast/from_int64_to_float32", "varShapes": [[2]], "varTypes": ["int64"],"varInit": ["uniform_int2"], "output": tf.float32},
+        #{"opName": "bitcast", "outName": "bitcast/emptyArrayTest/from_uint32_to_float32", "varShapes": [[0]],"varTypes": ["uint32"], "varInit": ["empty"], "output": tf.float32},
+
+        #{"opName": "bitcast", "outName": "bitcast/from_float32_to_float64", "varShapes": [[2]], "varTypes": ["float32"], "varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_bfloat16_to_float64", "varShapes": [[4]], "varTypes": ["bfloat16"],"varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_half_to_float64", "varShapes": [[4]], "varTypes": ["half"], "varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_float64_to_float64", "varShapes": [[1]], "varTypes": ["float64"], "varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_int32_to_float64", "varShapes": [[2]], "varTypes": ["int32"], "varInit": ["uniform_int2"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_int64_to_float64", "varShapes": [[2]], "varTypes": ["int64"], "varInit": ["uniform_int2"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/emptyArrayTest/from_uint32_to_float64", "varShapes": [[0]], "varTypes": ["uint32"], "varInit": ["empty"], "output": tf.float64},
```

Comment:
If I was doing it, to save time, I wouldn't write exhaustively every single combination of cases; I'd probably write 10-15 cases at most, that are representative (i.e., tests that vary over a bunch of different combinations shapes, ranks, datatypes, etc). Thorough is good, but there's diminishing returns eventually.
Adding empty array tests is great though. :+1: